apps/www/pages/maerzkampagne.js

```diff
@@ -421,59 +428,140 @@ ${pledgeLink}
-
-3. Wir sind von einer starken Community getragen. Von unseren Leserinnen lernen wir jeden Tag, wie wir besser werden können. Und über 1000 Freunde und Komplizen helfen mit, unseren Journalismus in jeden Winkel der Schweiz zu bringen.
-
-Damit die Republik in Zukunft bestehen kann, brauchen wir einen Wachstumsschub. Deshalb haben wir uns überlebenswichtige Ziele gesetzt: Bis Ende März mindestens 19’000 Mitglieder zu haben und 2,2 Millionen Franken zu finden.
-
-Erreichen wir die Ziele nicht, beenden wir das Projekt.
-
-Den grösseren Teil des Geldes haben wir durch die Grosszügigkeit unserer Mitglieder und Investorinnen bereits gefunden. Jetzt geht es darum, möglichst viele neue Leute von der Republik zu begeistern.
-
-Wenn Sie mitmachen und wir es nicht schaffen, bekommen Sie Ihr Geld zurück. Wenn wir es schaffen, bekommen Sie nicht nur vernünftigen Journalismus, sondern haben auch einen entscheidenden Beitrag zur Medienvielfalt in der Schweiz geleistet.
-
-${!activeMembership ? pledgeLink : ''}
-
-        `}
+        <>
+          <H2>Was ist die Republik?</H2>
+          <P>
+            Die Republik ist eine Dienstleistung für interessierte Menschen in
+            einer grossen, faszinierenden und komplexen Welt.
+          </P>
+          <P>
+            Wir recherchieren, fragen nach, ordnen ein und decken auf. Und
+            liefern Ihnen Fakten und Zusammenhänge als Grundlage für Ihre
+            eigenen Überlegungen und Entscheidungen.
+          </P>
+          <P>
+            Das ist eine heikle Aufgabe. Denn Journalismus ist alles andere als
+            harmlos: Es ist entscheidend, welche Geschichten erzählt werden.
+          </P>
+          <P>
+            Weil Vertrauen im Journalismus die härteste Währung ist, haben wir
+            die Republik so aufgestellt, dass wir diese Aufgabe für Sie
+            bestmöglich erledigen können:
+          </P>
+          <P>
+            **Wir sind unabhängig.** Und komplett werbefrei. So können wir uns
+            auf unseren einzigen Kunden konzentrieren: Sie. Und müssen weder
+            möglichst viele Klicks generieren noch Sie mit nervigen Anzeigen
+            belästigen. Und wir verkaufen Ihre persönlichen Daten niemals
+            weiter.
+          </P>
+          <P>
+            <strong>
+              Wir sind das transparenteste Medienunternehmen (das wir kennen).
+            </strong>
+            Wir legen alles offen: unsere Finanzen, Besitzverhältnisse,
+            Arbeitsweisen, Fehler, Löhne – weil wir überzeugt sind, dass es
+            wichtig ist zu zeigen, unter welchen Bedingungen Journalismus
+            hergestellt wird.
+          </P>
+
+          <P>
+            <strong style={{ ...fontStyles.serifBold }}>
```

Comment:
```suggestion
            <strong>
```
https://styleguide.republik.ch/typography#bold-and-cursive

if the font rule is missing in the legacy p component (checkable by verifying with dev tools that the bold font variant is used instead of faux bold) consider adding it to the component in the styleguide.